apps/website/src/data/tech/commands.ts

```diff
@@ -924,6 +929,32 @@ const commands: Record<string, Command> = {
     args: [],
   },
 
+  /**
+   * Text
+   */
+  text: {
+    description: "Sets the text to be displayed on stream",
+    category: "Text",
+    args: [
+      {
+        type: "string",
+        name: "text",
+        required: true,
+        variadic: false,
+      },
+    ],
+  },
+  showtext: {
+    description: "Shows text on stream",
```

Comment:
Out of curiosity - whats the difference between `text` and `showtext`?
Is `text` used for "queueing" the text that can later be displayed using `showtext`?

Reply:
That's my understanding, yeah. `text` sets the line of text to be displayed, `showtext` makes it appear on screen, and `hidetext` removes it from screen.